requirements.txt

```diff
@@ -1,60 +1,59 @@
-Flask==0.12.4
-Werkzeug==0.11.11
-Jinja2==2.8
-itsdangerous==0.24
-click==6.6
-MarkupSafe==0.23
-pyOpenSSL==17.5.0
-httplib2==0.10.3
+Flask==1.1.1
+Jinja2==2.10.3
+itsdangerous==1.1.0
+click==6.7
+MarkupSafe==1.1.1
+pyOpenSSL==19.0.0
+httplib2==0.14.0
 wtforms==2.2.1
-Flask-RESTful==0.3.5
-Flask-Login==0.4.0
+Flask-RESTful==0.3.7
+Flask-Login==0.4.1
 Flask-OAuthLib==0.9.5
 # pin this until https://github.com/lepture/flask-oauthlib/pull/388 is released
 requests-oauthlib>=0.6.2,<1.2.0
-Flask-SQLAlchemy==2.3.2
-Flask-Migrate==2.0.1
+Flask-SQLAlchemy==2.4.1
+Flask-Migrate==2.5.2
 flask-mail==0.9.1
-flask-talisman==0.6.0
+flask-talisman==0.7.0
 Flask-Limiter==0.9.3
-passlib==1.6.2
-aniso8601==1.1.0
-blinker==1.3
-psycopg2==2.7.3.2
+passlib==1.7.1
+aniso8601==8.0.0
+blinker==1.4
+psycopg2==2.8.3
 python-dateutil==2.8.0
-pytz==2016.7
-PyYAML==3.12
-redis==3.2.1
-requests==2.21.0
+pytz==2019.3
+PyYAML==5.1.2
+redis==3.3.11
+requests==2.22.0
 six==1.12.0
-SQLAlchemy==1.2.12
+SQLAlchemy==1.3.10
 # We can't upgrade SQLAlchemy-Searchable version as newer versions require PostgreSQL > 9.6, but we target older versions at the moment.
 SQLAlchemy-Searchable==0.10.6
 # We need to pin the version of pyparsing, as newer versions break SQLAlchemy-Searchable-10.0.6 (newer versions no longer depend on it)
 pyparsing==2.3.0
-SQLAlchemy-Utils==0.33.11
-sqlparse==0.2.4
-statsd==2.1.2
-gunicorn==19.7.1
+SQLAlchemy-Utils==0.34.2
+sqlparse==0.3.0
+statsd==3.3.0
+gunicorn==19.9.0
 rq==1.1.0
-rq-scheduler==0.9
+rq-scheduler==0.9.1
 celery==4.3.0
-kombu==4.6.3
-jsonschema==2.4.0
-RestrictedPython==3.6.0
-pysaml2==4.5.0
+kombu==4.6.5
+jsonschema==3.1.1
+RestrictedPython==5.0
+pysaml2==4.8.0
 pycrypto==2.6.1
-funcy==1.7.1
+funcy==1.13
 sentry-sdk>=0.12.2,<0.13.0
-semver==2.2.1
-xlsxwriter==0.9.3
+semver==2.8.1
+xlsxwriter==1.2.2
 pystache==0.5.4
-parsedatetime==2.1
-PyJWT==1.6.4
-cryptography==2.3
-simplejson==3.10.0
-ua-parser==0.7.3
-user-agents==1.1.0
+parsedatetime==2.4
+PyJWT==1.7.1
+cryptography==2.7
+simplejson==3.16.0
+ua-parser==0.8.0
+user-agents==2.0
 python-geoip-geolite2==2015.303
```

Comment:
Don't we need to replace this with the Python 3 version?

Reply:
Good catch, we should probably use `geoip2` instead.

Reply:
Done